plugins/modules/ag_listener.ps1

```diff
@@ -15,7 +15,7 @@ $spec = @{
     options = @{
         ag_name = @{type = 'str'; required = $true }
         listener_name = @{type = 'str'; required = $true }
-        ip_address = @{type = 'str'; required = $false }
+        ip_address = @{type = 'list'; required = $false }
```

Comment:
```suggestion
        ip_address = @{type = 'list'; elements='str'; required = $false }
```
Per the [sanity check](https://github.com/lowlydba/lowlydba.sqlserver/actions/runs/9006650802/job/25120714271?pr=246#step:4:935), the [type of elements](https://github.com/ansible/ansible/pull/66386#issuecomment-589717074) should be specified too (I assume this is the right syntax, but verify)